dev-docs/RFCs/v8.0/coordinate-spaces-rfc.md

```diff
@@ -0,0 +1,40 @@
+# RFC: Intuitive Coordinate Spaces
+
+* **Authors**: Tarek Sherif
+* **Date**: September 2019
+* **Status**: **Draft**
+
+
+## Summary
+
+This RFC proposes modifications to deck.gl's coordinate spaces to bring them closer to standard 3D model, view and clip spaces, making them easier to reason about and simplifying the implementation of algorithms that use them.
+
+## Overview
+
+deck.gl's [coordinate systems](https://deck.gl/#/documentation/developer-guide/coordinate-systems) include the following:
+- World space (model space in standard 3D) - Whatever space the data is defined in, often longitude/latitude or meter offsets
+- Common space (world space in standard 3D) - Primary shared space into which world coordinates are transformed, measured in mercator pixels. +x points to the right, +y points down (following the web mercator convention), and +z points above the map surface. Note that unlike the other 3D spaces in deck, this makes common space a left-handed coordinate system. This space also includes the mercator zoom, so it scales up as the zoom increases.
```

Comment:
we could move closer to WGS84 by defining common space in meter offsets?

Reply:
That could make sense. It would also have the nice side effect of letting us get rid of all the meter <-> pixel conversions. I'll add a note about that possibility.